### PR TITLE
Bug/#683

### PR DESCRIPTION
**Pull request by monsieurtanuki**
Fixed the calculation of bounding box zoom.

### CODE REVIEW

File: osmdroid-android/src/main/java/org/osmdroid/util/BoundingBox.java [hunk 1]
```diff
@@ -66,13 +66,28 @@ public BoundingBox concat(BoundingBox bb2){
...
 	public GeoPoint getCenter() {
 		return new GeoPoint((this.mLatNorth + this.mLatSouth) / 2.0,
 				(this.mLonEast + this.mLonWest) / 2.0);
 	}
 
+	/**
+	 * This version takes into consideration the date line
+	 * @since 5.6.6
+	 */
+	public GeoPoint getCenterWithDateLine() {
```
Comment (spyhunter99) [inline osmdroid-android/src/main/java/org/osmdroid/util/BoundingBox.java]: why not just put this code in get center?

Comment (monsieurtanuki) [reply osmdroid-android/src/main/java/org/osmdroid/util/BoundingBox.java]: I didn't want to break anything. I don't know who uses `getCenter`: maybe people now admit that `east` and `west` in `BoundingBox` only mean `longitude1` and `longitude2`, and populate them without any relevant order.

**Comment by spyhunter99** [comment]
hmm ok leave it as is then, makes sense

**Comment by spyhunter99** [comment]
can you merge master into this branch? it should clear up the travis failures

**Comment by monsieurtanuki** [comment]
Crashed again :(

**Pull request merged**